opencart 1.5.6.4/upload/admin/controller/module/smaily_for_opencart.php

```diff
@@ -63,8 +78,22 @@ public function index() {
         // Small texts.
         $this->data['small_subdomain'] = $this->language->get('small_subdomain');
         $this->data['small_password'] = $this->language->get('small_password');
+        $this->data['text_enabled'] = $this->language->get('text_enabled');
+        $this->data['text_disabled'] = $this->language->get('text_disabled');
         $this->data['token'] = $this->session->data['token'];
 
+        // Subscriber sync fields.
+        $this->data['customer_sync_enable_title'] = $this->language->get('customer_sync_enable_title');
+        $this->data['customer_sync_fields_title'] = $this->language->get('customer_sync_fields_title');
+        $this->data['customer_sync_fields_help'] = $this->language->get('customer_sync_fields_help');
+        // Subscriber sync cron fields.
+        $this->data['customer_sync_cron_token_title'] = $this->language->get('customer_sync_cron_token_title');
+        $this->data['customer_sync_cron_token_placeholder'] = $this->language->get('customer_sync_cron_token_placeholder');
+        $this->data['customer_sync_cron_token_help'] = $this->language->get('customer_sync_cron_token_help');
+        $this->data['customer_sync_cron_url'] = $this->config->get('config_url') . 'index.php?route=smailyforopencart/cron_customers&token=[token]';
```

Comment:
Where is this URL implemented?

Reply:
![Screenshot_2019-09-18 Smaily for Opencart](https://user-images.githubusercontent.com/33309474/65147566-186eb900-da27-11e9-8b2b-7803921bd49c.png)
The bold one.

Reply:
Sorry, I meant where is the function that handles route `smailyforopencart/cron_customers`?

Reply:
It's under catalog, I'll check the formatting and add it in the next commit.